src/main/java/org/almrangers/auth/aad/AadIdentityProvider.java

```diff
@@ -63,7 +63,7 @@ public AadIdentityProvider(AadSettings settings) {
     @Override
     public Display getDisplay() {
         return Display.builder()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)